docs/tests-ui.rst

```diff
@@ -12,6 +12,7 @@ Installing Dependencies
 
 3. From the `tests/ui/` directory, use NPM or another package manager to install Intern::
 
+    mkdir node_modules
```

Comment:
I'd prefer we left that out; `npm` is meant to create the directory if it's not there so you may need to configure your specific machine differently.